From e3ff4b33fd1a3d1ce3d1cdc0de665758bad2371f Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Mon, 5 Sep 2016 18:08:51 +0800
Subject: [PATCH] ARM: dts: rockchip: add sdcard io-domain node for miniarm

Both 3.3v and 1.8v are available on rk3288-miniarm board, so add sd
io-domain support in case need to deploy UHS speed mode.

Change-Id: I44a7c775620e32105c8c1b91219aac8b9d8b9e92
Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-miniarm.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-miniarm.dts b/arch/arm/boot/dts/rk3288-miniarm.dts
index 90c0e1a779ee..7c44b8cca3fd 100644
--- a/arch/arm/boot/dts/rk3288-miniarm.dts
+++ b/arch/arm/boot/dts/rk3288-miniarm.dts
@@ -111,6 +111,13 @@
 		regulator-boot-on;
 	};
 
+	io_domains: io-domains {
+		compatible = "rockchip,rk3288-io-voltage-domain";
+		rockchip,grf = <&grf>;
+
+		sdcard-supply = <&vccio_sd>;
+	};
+
 	/*
 	 * NOTE: vcc_sd isn't hooked up on v1.0 boards where power comes from
 	 * vcc_io directly.  Those boards won't be able to power cycle SD cards
-- 
2.35.3

